Dockerfile-build

```diff
@@ -1,6 +1,6 @@
 FROM node:8.11.1-alpine
 
-RUN apk add --no-cache git
+RUN apk add --no-cache git make gcc g++ linux-headers openssl python
```

Comment:
Sounds fine to me, but do we want to uninstall some of the build deps at the end in order to reduce the footprint of the final image?

Reply:
moved to https://github.com/mozilla/fxa-oauth-server/pull/563